src/extensions/askUserForParam.ts

```diff
@@ -6,15 +6,20 @@ const { marked } = require("marked");
 import { Param, ParamOption, ParamType } from "./extensionsApi";
 import * as secretManagerApi from "../gcp/secretManager";
 import * as secretsUtils from "./secretsUtils";
-import { logPrefix, substituteParams } from "./extensionsHelper";
+import { confirm, logPrefix, substituteParams } from "./extensionsHelper";
 import { convertExtensionOptionToLabeledList, getRandomString, onceWithJoin } from "./utils";
 import { logger } from "../logger";
 import { promptOnce } from "../prompt";
 import * as utils from "../utils";
 import { ParamBindingOptions } from "./paramHelper";
 
+enum SecretCreationAction {
+  CLOUD = 1,
```

Comment:
I think i might just be not understanding some nuance of TS enums, but why did you do this `CLOUD = 1, LOCAL,`? Does the `=1` set a default or something?

Reply:
it's just always better to start enum at 1 :)

Don't think this made it into the ts style but we have a firebase style guide that recommended this:
go/fireconsole-styleguide#always-start-enums-with-1